src/features/rewards/panelVerify/index.tsx

```diff
@@ -0,0 +1,100 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+import { getLocale } from '../../../helpers'
+
+import {
+  StyledWrapper,
+  StyledClose,
+  StyledHeader,
+  StyledBatIcon,
+  StyledHeaderText,
+  StyledTitle,
+  StyledSubtitle,
+  StyledListTitle,
+  StyledListItem,
+  StyledListIcon,
+  StyledListItemText,
+  StyledIDNotice,
+  StyledButton,
+  StyledFooter,
+  StyledFooterIcon
+} from './style'
+
+import {
+  CloseStrokeIcon,
+  UpholdColorIcon,
+  RewardsBATArrowsIcon,
+  RewardsCheckIcon
+} from '../../../components/icons'
+
+export interface Props {
+  onVerifyClick: () => void
+  onClose: () => void
+  id?: string
+}
+
+export default class PanelWelcome extends React.PureComponent<Props, {}> {
+  getListItem = (text: string) => (
+    <StyledListItem>
+      <StyledListIcon>
+        <RewardsCheckIcon />
+      </StyledListIcon>
+      <StyledListItemText>
+        {text}
+      </StyledListItemText>
+    </StyledListItem>
+  )
+
+  render () {
+    const {
+      onVerifyClick,
+      onClose,
+      id
+    } = this.props
+
+    return (
```

Comment:
Could we break this up in to a couple more methods? Maybe one for `footer` and `header`

Reply:
done